src/eduid_common/api/decorators.py

```diff
@@ -47,7 +48,7 @@ def _get_user():
 def require_user(f):
     @wraps(f)
     def decorated_function(*args, **kwargs):
-        user = _get_user()
+        user = get_dashboard_user()
```

Comment:
This will return a user document loaded in to the User class, see get_dashboard_user below.
Seems confusing that the function is called get_dashboard_user.